sdk/servicebus/service-bus/test/atomManagement.spec.ts

```diff
@@ -65,6 +65,215 @@ describe("Atom management - Namespace", function(): void {
+            managementTopic1,
+            managementSubscription1,
+            { name: baseName + "_rule_" + i }
+          )
+        ).name
+      );
+    }
+  });
+
+  after(async () => {
+    for (let i = 0; i < numberOfEntities; i++) {
+      await serviceBusAtomManagementClient.deleteQueue(baseName + "_queue_" + i);
+      await serviceBusAtomManagementClient.deleteTopic(baseName + "_topic_" + i);
+    }
+    await serviceBusAtomManagementClient.deleteTopic(managementTopic1);
+  });
+
+  function verifyEntities(methodName: string, receivedNames: string[]) {
+    let createdNames: string[];
+    if (methodName.includes("Queue")) {
+      createdNames = queueNames;
+    } else if (methodName.includes("Topic")) {
+      createdNames = topicNames;
+    } else if (methodName.includes("Subscription")) {
+      createdNames = subscriptionNames;
+    } else {
+      createdNames = ruleNames;
+    }
+    const numberOfReceived = receivedNames.length;
+    createdNames.forEach((createdName) => {
+      receivedNames = receivedNames.filter((receivedName) => createdName !== receivedName);
+    });
+    should.equal(
+      numberOfReceived,
+      receivedNames.length + createdNames.length,
+      "Unexpected number of entities received"
+    );
+  }
+
+  [
+    "getQueues",
+    "getQueuesRuntimeInfo",
+    "getTopics",
+    "getTopicsRuntimeInfo",
+    "getSubscriptions",
+    "getSubscriptionsRuntimeInfo",
+    "getRules"
+  ].forEach((methodName) => {
+    describe(`${methodName}`, () => {
+      it("Verify PagedAsyncIterableIterator", async () => {
+        const receivedEntities = [];
+        let iter = (serviceBusAtomManagementClient as any)[methodName]();
+        if (methodName.includes("Subscription")) {
+          iter = (serviceBusAtomManagementClient as any)[methodName](managementTopic1);
+        } else if (methodName.includes("Rule")) {
+          iter = (serviceBusAtomManagementClient as any)[methodName](
+            managementTopic1,
+            managementSubscription1
+          );
+        }
+        for await (const entity of iter) {
+          receivedEntities.push(
+            methodName.includes("Subscription") ? entity.subscriptionName : entity.name
+          );
+        }
+        verifyEntities(methodName, receivedEntities);
+      });
+
+      it("Verify PagedAsyncIterableIterator(generator .next() syntax)", async () => {
+        const receivedEntities = [];
+        let iter = (serviceBusAtomManagementClient as any)[methodName]();
+        if (methodName.includes("Subscription")) {
+          iter = (serviceBusAtomManagementClient as any)[methodName](managementTopic1);
+        } else if (methodName.includes("Rule")) {
+          iter = (serviceBusAtomManagementClient as any)[methodName](
+            managementTopic1,
+            managementSubscription1
+          );
+        }
+        let entityItem = await iter.next();
+        while (!entityItem.done) {
```

Comment:
no `for await`?

Reply:
I'll remove this test, the previous one is the same but with `for await`.